image/tools/entrypoint.sh

```diff
@@ -37,17 +37,19 @@ source "$DIR/lib/component/$component.sh"
 DATESTAMP=$(date '+%Y/%m/%d')
 DEST=/tmp/intly
 ARCHIVES_DEST=$DEST/archives
-mkdir -p $DEST $ARCHIVES_DEST
-export HOME=$DEST
+mkdir -p $DEST
 
 component_dump_data $DEST
 echo '==> Component data dump completed'
+
+export HOME=$DEST
```

Comment:
If $HOME is redefined before component_dump_data is called, you wanted be able to test locally because oc needs to access config in ~/.kube/config